### PR TITLE
tech(mock/json) #10 - Mock JSON(s) for the top keywords

### PR DESCRIPTION
**Pull request by shangan23**
Manually created JSON files from Google search page. Added it to the `resource` folder of the package.

Each JSON has 4 main keys,

1. total_results: Total number of results returned by the search result.
2. execution_time: Time is taken to fetch and display the result on the page in milliseconds.
3. query_string: Seach string
4. results: Array of results contains Title, URL and description for each result.

### CODE REVIEW

File: api/google-search/src/main/resources/amazon.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "total_results": 3990000000,
```
Comment (ramsunvtech) [inline api/google-search/src/main/resources/amazon.json:2]: `total_results` should be 6

File: api/google-search/src/main/resources/amazon.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    {
+      "title": "Welcome to Prime Video",
+      "url": "https://www.primevideo.com/",
+      "short_description": "Enjoy exclusive Amazon Originals as well as popular movies and TV shows. Watch anytime, anywhere. Start your free trial."
+    },
+    {
+      "title": "Amazon (@amazon) · Twitter",
+      "url": "https://twitter.com/amazon?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor",
+      "short_description": ""
```
Comment (ramsunvtech) [inline api/google-search/src/main/resources/amazon.json:24]: this part is story list. in future we have to revisit here

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: api/google-search/src/main/resources/amazon.json (hunk 1 above)
Comment (shangan23) [reply api/google-search/src/main/resources/amazon.json]: Updated all JSON(s) with the exact length of results.

File: api/google-search/src/main/resources/amazon.json (hunk 2 above)
Comment (shangan23) [reply api/google-search/src/main/resources/amazon.json:24]: sure

**Comment by naveenvignesh5** [comment]
Kindly squash the PR to have only 1 commit.

**Comment by shangan23** [comment]
> Kindly squash the PR to have only 1 commit.

Done.

**Comment by naveenvignesh5** [comment]
@shangan23 Kindly change the commit message according to `CONTRIBUTING.md`

**Review by naveenvignesh5 — APPROVED** [review]

**Review by yuvarajuv — APPROVED** [review]

**Comment by shangan23** [comment]
> @shangan23 Kindly change the commit message according to `CONTRIBUTING.md`

Done

**Review by ramsunvtech — APPROVED** [review]

**Review by jpukg — APPROVED** [review]

**Review by karthikeyanpa12 — APPROVED** [review]

**Pull request merged**